prow/cmd/deck/static/pr-script.js

```diff
@@ -821,7 +897,9 @@ function createPRCardBody(pr, builds, queries) {
     cardBody.appendChild(title);
     cardBody.appendChild(createJobStatus(builds));
     const nodes = pr.Labels && pr.Labels.Nodes ? pr.Labels.Nodes : [];
-    cardBody.appendChild(createMergeStatus(nodes, queries));
+    cardBody.appendChild(createMergeLabelStatus(nodes, queries));
+    const mergeable = pr.Mergeable ? pr.Mergeable : "UNKNOWN";
+    cardBody.appendChild(createMergeConflictStatus(mergeable, queries));
```

Comment:
Why do we need this and also to expose the needs-rebase label? For repos that don't turn on needs-rebase plugin?

Reply:
Yes, the needs-rebase plugin shouldn't be required by Tide.

Reply:
/shrug